src/Microsoft.VisualStudio.Services.Agent/ExtensionManager.cs

```diff
@@ -39,7 +39,7 @@ public sealed class ExtensionManager : AgentService, IExtensionManager
             switch (typeof(T).FullName)
             {
                 // Listener capabilities providers.
-                case "Microsoft.VisualStudio.Services.Agent.Listener.Capabilities.ICapabilitiesProvider":
+                case "Microsoft.VisualStudio.Services.Agent.ICapabilitiesProvider":
                     Add<T>(extensions, "Microsoft.VisualStudio.Services.Agent.Listener.Capabilities.AgentCapabilitiesProvider, Agent.Listener");
```

Comment:
should we move all capabilities providers to the common library?

Reply:
I was thinking the same. I started to make the code changes but the AgentCapabilitiesProvider uses something in Configuration so we would have to move all of those too since we don't want the shared library to take a dependency on the Listener. Or we would have Configuration in two places which might be confusing. Any ideas?

Reply:
Yep. i think that's just an unused using. :)

Reply:
Done.